styles.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,100&display=swap'); 
+flickity.css
+*
+{
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+}
+html{
+  scroll-behavior: smooth;
+}
+
+body
+
+{
+  margin: 0;
+  width: 100vw;
+  height: 100vh;
+  min-height: 20vh;
+  background:linear-gradient(#0b0c10,#1f2833);
+}
+
+
+div, section, span, ul, li, a, header{
+  box-sizing: border-box;
+}
+
+
+header
+
+{
+position: relative;
+width: 100%;
+padding: 10px 10px;
+display: flex;
+justify-content: space-between;
+align-items: center;
+z-index: 999;
+}
+
+header .logo
+{
+  color:whitesmoke;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 2em;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+}
+
+
+
+#menu li {
+  list-style: none;
+  display: inline-block;
+  margin: 20px;
+}
+
+#menu li a{
+  color:#0b0c10;
+  text-decoration: none;
+  font-size: 20px;
+}
+
+
+section
+{
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  float: left;
+}
+
+#home {
+  background-color: #212121;
+}
+
+#about {
+  background-color: #14FFEC;
+}
+
+#portfolio {
```

Comment:
portfolio and home have the same background color. You can do `#home, #portfolio {....}`